commands/create_cluster.go

```diff
@@ -178,6 +189,12 @@ func createClusterValidationOutput(cmd *cobra.Command, args []string) {
 		case IsConflictingFlagsError(err):
 			headline = "Conflicting flags used"
 			subtext = "When specifying a definition via a YAML file, certain flags must not be used."
+		case IsConflictingWorkerFlagsUsed(err):
```

Comment:
Kink of unrelated: I will need to use this error as well - is there a way to share headline / subtext right now? Otherwise I'll just have to copy it to scaling I guess.

Reply:
Copy it and we sort the merging later.